src/components/GameView/GameOverlays.vue

```diff
@@ -143,9 +155,13 @@ export default {
   display: flex;
   justify-content: center;
   align-items: center;
+  text-align: center;
+  font-family: 'PT Serif', 'serif';
```

Comment:
I still don't think we should be doing this, this should be a build in text class in Vuetify somewhere and not a one off here. This also feels like a follow up improvement to me since it will likely involve modifying the build system to include or modify a sass config file from vuetify however. cc @itsalaidbacklife

Reply:
I've removed the `font-family` style as it's now unneeded in this file.

Reply:
Because this is now globally configured with the scss var, right?

Reply:
Yep, the app should be using that font family var.

Reply:
@TommySchmid it looks like we're still setting the font family on line 159 here. We should be safe to remove this now that the Vuetify theme is setup

Reply:
Removed, in addition to `font-family` references in `App.vue` and `StatsView.vue`.